src/components/Image.js

```diff
@@ -109,16 +115,23 @@ export const Image = ({
         resizeMode={resizeMode}
         borderRadius={borderRadius}
       >
-        {!!message && <ImageMessage message={message} />}
-        {!!globalSettings?.showImageRights && !!sourceProp?.copyright && (
-          <ImageRights imageRights={sourceProp.copyright} />
+        {showChildren && (
+          <View style={styles.contentContainerStyle}>
+            {!!message && <ImageMessage message={message} />}
+            {!!button && <ImageButton button={button} />}
+            {!!showImageRights && <ImageRights imageRights={sourceProp.copyright} />}
```

Comment:
like in `const showChildren = !!message || !!button || showImageRights;` you do not need the !! for the boolean here

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/957/commits/4879aaf1097c8856e567a9ae1ffd90d32d17d8cc